tinker/faculty_bios/__init__.py

```diff
@@ -38,54 +38,54 @@ def before_request(self, name, **kwargs):
 
     def index(self):
         username = session['username']
+        return self.index_cache(username)
 
-        # I removed cacheing, because it was taking too long for people to see the "activate/deactivate" changes - caleb
-        # @cache.memoize(timeout=600)
-        def index_cache(username):
-            roles = session['roles']
+    @cache.memoize(timeout=600)
+    def index_cache(self, username):
```

Comment:
I would put the index_cache method inside the index method. It helps for readability

Reply:
i needed to move the cache method outside of the index method because I could't reach the index cache method from outside of the regular index method

Reply:
as for your question, the only loads that are 'slow' are when the page isn't cached at all.

Reply:
gotcha, that makes sense 👍